### ISSUE TITLE
Single Page Implementation

### ISSUE THREAD

**Issue by ryanwake01** [Diyzzuf/grav-plugin-private#3]
I am a little confused. I need to implement this on a single page. From what I am reading in the ReadMe file, I should set private_site: false and add private-tag: hidden to the yaml on the page I am trying to make private? Or am I getting this wrong? Would it be possible for you to provide a sample private.yaml file that targets one page specifically?

**Comment by Diyzzuf** [comment]
Sorry, i can't check my mail these days.
For a single private page ( or more but not a fully private site ) : 

On config.yaml

```
private_site: false 
private_tag: hidden   //  hidden or what you whant
```

and on yourpage/page.md for exemple

```
---
title: Title of my page
date: 17:34 03/27/2015 
taxonomy:
    category: blog, tuto
    tag: [tuto, hidden]  // Add your hidden tag here
---

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec ultricies tristique nulla et mattis. Phasellus id massa eget nisl congue blandit sit amet id ligula. Praesent et nulla eu augue tempus sagittis. Mauris faucibus nibh et nibh cursus in vestibulum sapien egestas. Curabitur ut lectus tortor. Sed ipsum eros, egestas ut eleifend non, elementum vitae eros. Mauris felis diam, pellentesque vel lacinia ac, dictum a nunc. Mauris mattis nunc sed mi sagittis et facilisis tortor volutpat. Etiam tincidunt urna mattis erat placerat placerat ac eu tellus.



```

**Comment by mexomagno** [comment]
This simply doesn't work for me.
I can only use this plugin with "private_site: true". When I set it to false, the Private plugin just wont show on any of the pages where I set the private tag.
Just to be clear, this is my setup:
`user/config/plugins/private.yaml:`

```
    private_site: false
    private_tag: hidden
```

`user/pages/XX.testpage/default.md:`

```
    ---
    title: TestPage
    menu: Testpage
    taxonomy:
        tag: [hidden]
    ---
```

`user/config/site.yaml:`

```
    taxonomies: [tag]
```

Also, it's worth saying that i'm using lighttpd and I managed to get Grav working by adding this line to lighttpd.conf:
`url.rewrite-if-not-file = ( "^/(?!(otros)).*$" => "/index.php")`
(it must ignore the "otros" folder and it's files) I dont know if that's enough.

Any help will be greatly appreciated.
